app/modules/asset_groups/models.py

```diff
@@ -412,16 +413,24 @@ def detected(self, job_id, response):
         if job is None:
             raise HoustonException(log, f'job_id {job_id} not found')
 
-        status = response.get('status')
+        status = message.get('status')
         if not status:
             raise HoustonException(log, 'No status in response from Sage')
 
-        if status != 'completed':
+        success = status.get('success')
```

Comment:
Most of the changes in `AssetGroup.detected` need to be reverted. @hwindsor made these changes without knowing the actual response from sage.